src/lib/client.js

```diff
@@ -160,4 +165,25 @@ class Client extends EventEmitter {
   }
 }
 
+function * getQuote (connector, query) {
+  try {
+    const res = yield request.get(connector + '/quote').query(query)
+    return res.body
+  } catch (err) {
+    if (err.response.body.id !== 'AssetsNotTradedError') throw err
+  }
+}
+
+function getCheaperQuote (quote1, quote2) {
+  if ((new BigNumber(quote1.source_amount))
+      .lessThan(quote2.source_amount)) {
+    return quote1
+  }
+  if ((new BigNumber(quote1.destination_amount))
```

Comment:
This doesn't make sense to me. If
`quote1` is 5 USD -> 4.2 EUR,
`quote2` is 5 USD -> 4.5 EUR,
this will return `quote1`, but the "cheaper" quote should be the second, right?